fastlane/Fastfile

```diff
@@ -705,9 +705,9 @@ REPOSITORY_NAME = 'WordPress-Android'
     values = options[:version].split('.')
     files = {
       "release_note_#{values[0]}#{values[1]}" => { desc: "changelogs/#{options[:build_number]}.txt", max_size: 500, alternate_key: "release_note_short_#{values[0]}#{values[1]}" },
+      play_store_app_title: { desc: 'title.txt', max_size: 30 },
       play_store_promo: { desc: 'short_description.txt', max_size: 80 },
-      play_store_desc: { desc: 'full_description.txt', max_size: 0 },
-      play_store_app_title: { desc: 'title.txt', max_size: 30 }
+      play_store_desc: { desc: 'full_description.txt', max_size: 0 }
```

Comment:
ℹ️  This is just my OCD reordering the list of files so that title.txt comes before `short_description` and `full_description` instead of last 😅  (and ditto for the corresponding Jetpack lane on L736-L738)